borg/archiver.py

```diff
@@ -1174,12 +1173,12 @@ def parse_args(self, args=None):
             args = self.preprocess_args(args)
         parser = self.build_parser(args)
         args = parser.parse_args(args or ['-h'])
+        self.args = args
```

Comment:
oh, i remember now: by passing args in its totality, i do not need to refer to `changed` directly, which is defined only for certain subcommands. but okay, i moved it to `run()` and used getattr to fallback to proper defaults.